DoSomethingArchive/legacy-website
# Address validation

#### What's this PR do?

Fixes bug on `/user/:uid/info` form that validates address correctly and doesn't throw an error if address is not present (since it is not required). 
#### How should this be manually tested?

Visit  `/user/:uid/info` and:
- delete address - should save profile. 
- add address and input `z` as the zipcode - should throw error. 
#### What are the relevant tickets?

Fixes #6317
